### PR TITLE
USHIFT-356: MicroShift: skip sig-node tests

### PR DESCRIPTION
**Pull request by pacevedom**

### CODE REVIEW

File: test/extended/machines/cluster.go [hunk 1]
```diff
@@ -79,6 +79,10 @@ var _ = g.Describe("[sig-node] Managed cluster", func() {
 	})
 
 	g.It("should report ready nodes the entire duration of the test run [Late]", func() {
+		_, _, _, prometheusAvailable := prometheus.LocatePrometheus(oc)
+		if !prometheusAvailable {
+			g.Skip("No prometheus available")
```
Comment (ingvagabund) [inline test/extended/machines/cluster.go:84]: @rphillips @joelsmith for awareness

We can't rely on Prometheus getting installed in every cluster. @rphillips is there any other way of detecting whether all the nodes were ready?

Comment (ingvagabund) [reply test/extended/machines/cluster.go:84]: @rphillips is this test supposed to be gating? Or, is it safe to skip it when Prometheus is not installed?

Comment (pacevedom) [reply test/extended/machines/cluster.go]: Swapped the prometheus check with apigroups belonging to the cluster monitoring operator.

File: test/extended/machines/cluster.go [hunk 2]
```diff
@@ -78,7 +78,7 @@ var _ = g.Describe("[sig-node] Managed cluster", func() {
 		}
 	})
 
-	g.It("should report ready nodes the entire duration of the test run [Late]", func() {
+	g.It("should report ready nodes the entire duration of the test run [Late][apigroup:monitoring.openshift.io][apigroup:monitoring.coreos.com]", func() {
```
Comment (ingvagabund) [inline test/extended/machines/cluster.go:81]: @pacevedom would you please update the PR description to mention why both apigroups are needed?

Comment (pacevedom) [reply test/extended/machines/cluster.go:83]: Done

**Comment by haircommander** [comment]
LGTM (node team)

File: test/extended/machines/cluster.go [hunk 3]
```diff
@@ -78,7 +78,9 @@ var _ = g.Describe("[sig-node] Managed cluster", func() {
 		}
 	})
 
-	g.It("should report ready nodes the entire duration of the test run [Late]", func() {
+	// This test makes use of Prometheus metrics, which are not present in the absence of cluster-monitoring-operator, the owner for
+	// the api groups tagged here.
+	g.It("should report ready nodes the entire duration of the test run [Late][apigroup:monitoring.openshift.io][apigroup:monitoring.coreos.com]", func() {
 		// we only consider samples since the beginning of the test
```
Comment (ingvagabund) [inline test/extended/machines/cluster.go:84]: @pacevedom `[apigroup:monitoring.openshift.io]` should be dropped

**Pull request merged**